docs/en/stack/security/securing-communications/tutorial-tls-intro.asciidoc

```diff
@@ -27,7 +26,16 @@ described in
 particular, this tutorial provides instructions that work with the `zip` and
 `tar.gz` packages.
 
-. <<get-started-license,Install a trial license>>.
+. Verify that you are using a license that includes the encrypted communications
```

Comment:
Same comment as previous - this paragraph and the following one aren't really necessary if they are using the default distribution, as there are no license types for which TLS isn't supported